package.json

```diff
@@ -29,56 +29,58 @@
     "@arangodb": false
   },
   "devDependencies": {
-    "@types/ansi-styles": "^3.2.0",
-    "@types/chai": "^4.1.2",
-    "@types/chai-as-promised": "^7.1.0",
-    "@types/cors": "^2.8.1",
+    "@types/ansi-styles": "^3.2.1",
+    "@types/chai": "^4.2.3",
+    "@types/chai-as-promised": "^7.1.2",
     "@types/deep-equal": "^1.0.1",
-    "@types/graphql": "~14.2.0",
+    "@types/deep-equal-in-any-order": "^1.0.1",
+    "@types/graphql": "^14.5.0",
```

Comment:
`@types/graphql` is no longer needed (it's only [a stub](https://www.npmjs.com/package/@types/graphql) now), so this can be removed

Reply:
agreed

Reply:
Shouldn't it be removed from peerDependencies as well?

```
  "peerDependencies": {
    "@types/graphql": "^14.0.0",
    "graphql": "^14.0.0"
  },
```

Reply:
ah yes, exactly 👍

Reply:
ok, I'll fix